components/sync15/src/state.rs

```diff
@@ -80,7 +81,7 @@ pub struct GlobalState {
     pub collections: InfoCollections,
     pub global: MetaGlobalRecord,
     pub global_timestamp: ServerTimestamp,
-    pub keys: CollectionKeys,
+    pub keys: EncryptedBso,
```

Comment:
The comment on this struct says "the default encryption key" and may be slightly out of date or confusing now. May be worth re-wording to say that we keep the encrypted c/k blob, and mention that we keep it encrypted as an extra check to ensure we are syncing with the right root key.